neo/Network/P2P/Payloads/Transaction.cs

```diff
@@ -130,16 +130,14 @@ public UInt160[] GetScriptHashesForVerifying(Snapshot snapshot)
             return hashes.OrderBy(p => p).ToArray();
         }
 
-        public virtual bool Reverify(Snapshot snapshot, IEnumerable<Transaction> mempool)
+        public virtual bool Reverify(Snapshot snapshot, BigInteger currentFee)
```

Comment:
`currentFee` -> `currentSenderTotalFee` ?

Reply:
But `currentFee` looks good as well, however, I prefer a little bit more explanatory here.
When I looked I needed to double check, with a better naming it will be quite clear.

Reply:
current fee means sum of systemFee & networkFee of all transactions in mempool from a specified sender, maybe there is better name for that:)

Reply:
It is good but not so good...aehauheuaea
Because it gives the idea that it is the `currentFee` of the transaction.

I will rename it to `totalSenderFeesFromPool`

`Pool` looks good because it is more generic to any set we want to compare against.